src/main/java/com/iexec/worker/tee/post/compute/utils/EnvUtils.java

```diff
@@ -18,8 +18,7 @@ public static String getEnvVar(String envVarName) {
     public static String getEnvVarOrThrow(String envVarName, ReplicateStatusCause statusCauseIfMissing) throws PostComputeException {
         String envVar = System.getenv(envVarName);
         if (envVar == null || envVar.isEmpty()) {
-            log.error("Required env var is empty [name:{}]", envVarName);
-            throw new PostComputeException(statusCauseIfMissing);
+            throw new PostComputeException(statusCauseIfMissing, "Required env var is empty [name:" + envVarName + "]");
```

Comment:
Adding a message to the exception is great, but don't you think it nice for the administrator to also keep error logging?

Reply:
Errors are logged there:
https://github.com/iExecBlockchainComputing/tee-worker-post-compute/blob/286f492b0138f0f0c6258e2ddeb61c3711f43860/src/main/java/com/iexec/worker/tee/post/compute/PostComputeAppRunner.java#L60

Reply:
It was already there, thanks :)